html/changelogs/Tigercat2000-tgcams.yml

```diff
@@ -0,0 +1,41 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes: 
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   spellcheck (typo fixes)
+#   experiment
+#################################
+
+# Your name. Remove the quotation mark and put in your name when copy+pasting the example changelog.
+author: Tigercat2000
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - bugfix: "Fixed a bug where the AI could not delete photos it takes."
+  - rscadd: "Ported -tg- cameranets. This means that all of our camera based systems are more or less up to date with -tg-; 510 features are missing."
+  - rscadd: "An AI with the 'camera lights' mode on will now, instead of the button toggling the closest camera's light, toggle camera lights on/off as the AI moves."
+  - rscadd: "It now takes a minimum of 30 deciseconds between attempting to track someone, and locking on. This number is increased by their distance away from the AI eye."
+  - tweak: "The camera activate/deactivate proc is now called 'toggle_cam'."
+  - tweak: "The 'trackable' proc has been replaced by a more object-oriented 'mob.can_track()' proc."
+  - tweak: "The 'networks' section of the security camera console is now above the camera list."
```

Comment:
.....yes. Rule 531 of UI design, things that change the height of the page should always be available at wherever the page resets the scroll to when the height changes.